Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -368,6 +370,13 @@
         'Era' : 'Phase2C9_timing_layer_bar',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFullTrigger','RecoFullGlobal', 'HARVESTFullGlobal'],
     },
+    '2026D47' : {
+        'Geom' : 'Extended2026D47',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic',
```

Comment:
Hmmm there is an issue here, no?

You need to add: 
'auto:phase2_realistic_T15'

See https://github.com/cms-sw/cmssw/pull/27517/ from @mmusich

Reply:
yes this needs to be changed, but as it was not clear this PR was going to be merged at all I didn't comment.

Reply:
Corrected that part now